test/e2e/setup.go

```diff
@@ -23,12 +23,12 @@ import (
 	"github.com/tendermint/tendermint/types"
 )
 
-type GenesisAccount struct {
+type Account struct {
 	PubKey        cryptotypes.PubKey
 	InitialTokens int64
 }
 
-func MakeGenesis(nodes []*Node, accounts []*GenesisAccount) (types.GenesisDoc, error) {
+func MakeGenesis(nodes []*Node, accounts []*Account) (types.GenesisDoc, error) {
```

Comment:
Do you think this function could use the `genesis` package instead of duplicating it?

Reply:
yeah definitely! was going to do that in a follow up allow with testground and testapp